### PR TITLE
[Student][MBL-13648] DST fix for calendar events in syllabus

### PR DESCRIPTION
**Pull request by JordanMarshall**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1014595&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/student/src/main/java/com/instructure/student/mobius/syllabus/SyllabusPresenter.kt [hunk 1]
```diff
@@ -69,8 +69,10 @@ object SyllabusPresenter : Presenter<SyllabusModel, SyllabusViewState> {
     private fun getDateString(event: ScheduleItem, context: Context): String {
         return if (event.startAt == null) {
             context.getString(R.string.toDoNoDueDate)
-        } else {
+        } else if (event.itemType == ScheduleItem.Type.TYPE_ASSIGNMENT){
             event.startAt!!.toDueAtString(context)
```
Comment (Blackchai) [inline apps/student/src/main/java/com/instructure/student/mobius/syllabus/SyllabusPresenter.kt]: Can we add a test case to the presenter that uses startDate?

Comment (JordanMarshall) [reply apps/student/src/main/java/com/instructure/student/mobius/syllabus/SyllabusPresenter.kt]: `startDate` is a getter that derives its value from `startAt`, which is already being used in the tests

Comment (Blackchai) [reply apps/student/src/main/java/com/instructure/student/mobius/syllabus/SyllabusPresenter.kt]: Ahh I see, my bad.

**Pull request merged**